app/src/main/java/org/commcare/dalvik/reminders/sync/CommCareUtils.kt

```diff
@@ -38,7 +44,12 @@ object CommCareUtils {
                 val time = caseProperties[TIME]
                 val sessionEndpoint = caseProperties[SESSION_ENDPOINT] ?: ""
                 val caseId = caseProperties[CASEID] ?: ""
+                val usersToSkipReminder = caseProperties[USERS_TO_SKIP_REMINDER] ?: ""
 
+                if (currentUserId != null && usersToSkipReminder.isNotEmpty()
+                    && usersToSkipReminder.contains(currentUserId, true)){
+                    continue
+                }
```

Comment:
`usersToSkipReminder.isNotEmpty()` seems to be redundant with `usersToSkipReminder.contains(currentUserId, true)`

Reply:
This is just to avoid unnecessary checks as `isNotEmpty` only checks the length of the `String`